scripts/build.py

```diff
@@ -163,3 +163,8 @@ def get_cache_slug():
     CSS_SRC_DIR = os.path.join('core', 'templates', 'prod', 'head', 'css', '')
     CSS_OUT_DIR = os.path.join(BUILD_DIR, 'css', '')
     copy_files_source_to_target(CSS_SRC_DIR, CSS_OUT_DIR)
+
+    # Copy core/templates/prod/head/ to build/[cache_slug]/templates/head/
+    TEMPLATES_SRC_DIR = os.path.join('core', 'templates', 'prod', 'head', '')
+    TEMPLATES_OUT_DIR = os.path.join(BUILD_DIR, 'templates', 'head', '')
```

Comment:
This should be separate work. It sounds fine to me, since it will make moving the templates to static html files a bit easier in the future.  That being said, I think it should just be a detailed issue and not addressed during GSoC.